zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/cacheAPI/SendNotificationCacheApi.java

```diff
@@ -0,0 +1,43 @@
+package com.github.onsdigital.zebedee.model.publishing.cacheAPI;
+
+import com.github.davidcarboni.httpino.*;
+import com.github.onsdigital.zebedee.configuration.Configuration;
+import com.github.onsdigital.zebedee.json.EventType;
+import com.github.onsdigital.zebedee.model.publishing.PublishNotification;
+import com.github.onsdigital.zebedee.model.publishing.WebsiteResponse;
+import com.github.onsdigital.zebedee.util.EncryptionUtils;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+
+import java.io.IOException;
+import java.util.List;
+
+import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+
+public class SendNotificationCacheApi {
+    private static final String LEGACY_CACHE_API_PATH = "/v1/cache-times/";
+
+    public SendNotificationCacheApi() {
```

Comment:
We don't send notifications to the Legacy Cache API, so this should be renamed. Maybe something like `LegacyCacheApiClient`, for instance. And then you can rename `sendNotification` to `sendPayloads`.